examples/advanced/tidb-cluster.yaml

```diff
@@ -134,6 +134,10 @@ spec:
   ## If specified, the pods will be scheduled by the specified scheduler.
   ## Can be overwritten by component settings.
   schedulerName: tidb-scheduler
+  
+  ## PodManagementPolicy default Parallel.
+  ## Ref: https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#pod-management-policies
+  podManagementPolicy: Parallel
```

Comment:
Do we need to change the default value as OrderedReady?
@DanielZhangQD

Reply:
We need to keep the current behavior for the default configuration.

Reply:
ok, for pump it is not `keep the current behavior for the default configuration`, it does not set the value before